tests/e2e/pageobjects/ide/DialogWindow.ts

```diff
@@ -0,0 +1,86 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { injectable, inject } from 'inversify';
+import { CLASSES } from '../../inversify.types';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { By } from 'selenium-webdriver';
+import { Logger } from '../../utils/Logger';
+import { TestConstants } from '../../TestConstants';
+
+@injectable()
+export class DialogWindow {
+    private static readonly DIALOG_BODY_XPATH_LOCATOR: string = '//div[@id=\'theia-dialog-shell\']//div[@class=\'dialogBlock\']';
+    private static readonly CLOSE_BUTTON_XPATH_LOCATOR: string = `${DialogWindow.DIALOG_BODY_XPATH_LOCATOR}//button[text()='close']`;
+
+    constructor(@inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async dialogDisplayes(): Promise<boolean> {
+        Logger.debug('WarningDialog.dialogDisplayes');
```

Comment:
Please replace all WarningDialog with DialogWindow in this class.

Reply:
Great catch, thank you.